recipes/mdnsd/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+import os
+import sys
+from conans import ConanFile, CMake, tools
+from conans.tools import download, unzip
+from conans.model.version import Version
+
+
+class mdnsdConan(ConanFile):
+    name = "mdnsd"
+    license = "BSD 3-Clause"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/**"
+    ]
+    homepage = "https://github.com/Pro/mdnsd"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Improved version of Jeremie Miller's MDNS-SD implementation"
+    topics = ("dns", "daemon", "multicast", "embedded", "c")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "cpp_compatible": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+        "cpp_compatible": False
+    }
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        if Version(self.version) <= "0.8.0":
+            folder_name = "%s-%s" % (self.name, "0.8")
+        else:
+            folder_name = "%s-%s" % (self.name, self.version)
+        os.rename(folder_name, self._source_subfolder)
+
+    def config_options(self):
+        if not self.options.cpp_compatible:
+            del self.settings.compiler.libcxx
+            del self.settings.compiler.cppstd
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.verbose = True
```

Comment:
```suggestion
```

Reply:
is there a particular need to remove verbosity for cmake? Wouldn't it make it easier to analyze logs when something goes wrong?

Reply:
You're completely right that it makes it easier to analyze logs.
I feel like this should be configurable globally by the cmake build helper.
Right now it is recipe per recipe: https://github.com/conan-io/conan/blob/1df93ed3d14c48e534d9c7842ba9385f35253678/conans/client/build/cmake.py#L347-L357

Reply:
I've left a comment at https://github.com/conan-io/conan/issues/5765